app/lib/signup/domains_builder.rb

```diff
@@ -15,7 +15,7 @@ def generate
     end
 
     def generate_subdomain
-      result = org_name.to_s.parameterize
+      result = org_name.to_s.tr('_', '-').parameterize
 
       while result.present? && invalid_subdomain_condition.call(subdomain: result)
         base, sequence = result.match(/\A(.*?)(?:\-(\d+))?\z/).to_a[1, 2]
```

Comment:
should we use `Authentication::RE_DOMAIN_OK` instead?

Reply:
Not sure how different these regexes are, but I think we don't need to touch it in this PR.

Reply:
I didn't think much before writing. Yeah, it has some grouping and stuff that obviously will not work OOB. So we can merge with this.